node/subsystem/src/messages.rs

```diff
@@ -51,6 +51,14 @@ pub enum CandidateSelectionMessage {
 	Invalid(Hash, CandidateReceipt),
 }
 
+impl CandidateSelectionMessage {
+	pub fn hash(&self) -> Option<Hash> {
```

Comment:
needs docs

Reply:
and I think `hash` is not a great name for this. It should be `relay_parent`, I'd assume